src/constants/LayoutConstants.js

```diff
@@ -10,15 +10,31 @@ export const COLORS = {
 
 export const DISABLED_COLORS = ['#e0e0e0', '#cacaca', '#a8a8a8', '#8d8d8d', '#6f6f6f'];
 
-export const CARD_SIZES = {
+export const LEGACY_CARD_SIZES = {
   XSMALL: 'XSMALL',
   XSMALLWIDE: 'XSMALLWIDE',
-  SMALL: 'SMALL',
   TALL: 'TALL',
-  MEDIUM: 'MEDIUM',
   WIDE: 'WIDE',
-  LARGE: 'LARGE',
   XLARGE: 'XLARGE',
+  SMALL: 'SMALL',
```

Comment:
You can just spread the CARD_SIZES object here... instead of repeating the constants